packages/opentelemetry-api/src/context/context.ts

```diff
@@ -38,49 +36,45 @@ const SUPPRESS_INSTRUMENTATION_KEY = createContextKey(
 const BAGGAGE_KEY = createContextKey('OpenTelemetry Baggage Key');
 
 /**
- * Return the active span if one exists
+ * Return the span if one exists
  *
  * @param context context to get span from
  */
-export function getActiveSpan(context: Context): Span | undefined {
-  return (context.getValue(ACTIVE_SPAN_KEY) as Span) || undefined;
+export function getSpan(context: Context): Span | undefined {
+  return (context.getValue(SPAN_KEY) as Span) || undefined;
 }
 
 /**
- * Set the active span on a context
+ * Set the span on a context
  *
  * @param context context to use as parent
  * @param span span to set active
  */
-export function setActiveSpan(context: Context, span: Span): Context {
-  return context.setValue(ACTIVE_SPAN_KEY, span);
+export function setSpan(context: Context, span: Span): Context {
+  return context.setValue(SPAN_KEY, span);
 }
 
 /**
- * Wrap extracted span context in a NoopSpan and set as active span in a new
+ * Wrap span context in a NoopSpan and set as span in a new
  * context
  *
  * @param context context to set active span on
  * @param spanContext span context to be wrapped
  */
-export function setExtractedSpanContext(
+export function setSpanContext(
```

Comment:
This looks like a side effect issue waiting to happen and as I can't any usages in the opentelemetry-js repo, can I suggest that we remove it...

As a newbie I can see someone calling this and then attempting to set the attributes or something on the context which would all effectively fail silently.

Failing that perhaps renaming this to include or specify that the span becomes effectively read only?

Reply:
`setSpanContext()` is just a thin helper on top of `setSpan()`. We could remove `setSpan()` by exporting `SPAN_KEY` instead. Because context keys are actually symbols stored in the global symbol registry it would be even work to just document to use `OpenTelemetry Context Key SPAN` and don't export anything.
IMHO exporting `setSpan()`/`getSpan()` is by far easier to understand and use.

Setting something on a `Context` is never replacing anything because it's always a new `Context` which is returned.